### ISSUE TITLE
Filter patients in vignettes so that they render quicker on the site

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/admiral#1265]
### Background Information

See the approach used in https://github.com/pharmaverse/admiral/blob/main/vignettes/bds_finding.Rmd as follows:
```{r echo=FALSE}
vs <- filter(vs, USUBJID %in% c("01-701-1015", "01-701-1023", "01-703-1086", "01-703-1096", "01-707-1037", "01-716-1024"))
```

Similar needs to be added to restrict the patients in the main input datasets contributing to the following vignettes as well:
https://github.com/pharmaverse/admiral/blob/main/vignettes/adsl.Rmd
https://github.com/pharmaverse/admiral/blob/main/vignettes/bds_tte.Rmd
https://github.com/pharmaverse/admiral/blob/main/vignettes/occds.Rmd

### Definition of Done

All the dataset_vignette calls should as a result run quicker and then the webpages like https://pharmaverse.github.io/admiral/articles/occds.html will render quicker.

**Comment by thomas-neitmann** [comment]
I think we discussed at one point to limit the number of patients already in {admiral.test} itself. Looks like a cleaner approach compared to filtering in every single vignette.

**Comment by thomas-neitmann** [comment]
Alright, fine with me 👍

**Comment by thomas-neitmann** [comment]
Can we use this for the "hackR"?

**Comment by PuzzledFace** [comment]
@thomas-neitmann @rossfarrugia Do you want to restrict vignettes to a specific number of participants or to specific patient IDs?  

[https://github.com/pharmaverse/admiral/blob/main/vignettes/adsl.Rmd](https://github.com/pharmaverse/admiral/blob/main/vignettes/adsl.Rmd) renders almost instantly when I knit locally and involves only 10 patients and there seems to be an appropriate amount of heterogeneity amongst their data.

**Comment by PuzzledFace** [comment]
After discussion with @rossfarrugia, we agreed NOT to filter example data for [https://github.com/pharmaverse/admiral/blob/main/vignettes/adsl.Rmd](https://github.com/pharmaverse/admiral/blob/main/vignettes/adsl.Rmd).  ADSL is one-record-per-participant, so the vignette renders quickly (sample data frames include only ten observations at most) and filtering removes examples of functional features that are discussed in text.

**Comment by bms63** [comment]
Hi @PuzzledFace We leave issues open until a release is pushed out.  If you PR title has close #1273 in it then it will auto-close the issue once devel is merged into main
